### PR TITLE
Remove unused debug writelns in std.algorithm

### PR DESCRIPTION
**Pull request by JackStouffer**
These didn't even work because std.stdio wasn't imported. I highly doubt anyone was using them, and they're not particularly useful anyway.

### CODE REVIEW

File: std/algorithm/mutation.d [hunk 1]
```diff
@@ -1871,18 +1850,15 @@ if (s == SwapStrategy.stable
...
         [ 0, 2, 3, 4, 6, 7, 8, 9, 10 ]);
 
     a = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
-    //writeln(remove!(SwapStrategy.stable)(a, 1, 3, 5));
     a = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
     assert(remove!(SwapStrategy.stable)(a, 1, 3, 5)
             == [ 0, 2, 4, 6, 7, 8, 9, 10]);
     a = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
-    //writeln(remove!(SwapStrategy.stable)(a, 1, tuple(3, 5)));
     a = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
```
Comment (wilzbach) [inline std/algorithm/mutation.d]: This line can then be removed as well?

File: std/algorithm/mutation.d [hunk 2]
```diff
@@ -1871,18 +1850,15 @@ if (s == SwapStrategy.stable
...
 
     a = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
-    //writeln(remove!(SwapStrategy.stable)(a, 1, 5));
     a = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
     assert(remove!(SwapStrategy.stable)(a, 1, 5) ==
         [ 0, 2, 3, 4, 6, 7, 8, 9, 10 ]);
 
     a = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
-    //writeln(remove!(SwapStrategy.stable)(a, 1, 3, 5));
     a = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
```
Comment (wilzbach) [inline std/algorithm/mutation.d]: This line can then be removed as well?

File: std/algorithm/mutation.d [hunk 3]
```diff
@@ -1871,18 +1850,15 @@ if (s == SwapStrategy.stable
            [ 1, 2, 4 ]);
 
     a = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
-    //writeln(remove!(SwapStrategy.stable)(a, 1, 5));
     a = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
```
Comment (wilzbach) [inline std/algorithm/mutation.d]: This line can then be removed as well?

**Comment by andralex** [comment]
That code was mightily useful back in the days when dmd had more bugs than features :)

**Comment by JackStouffer** [comment]
Auto-merge toggled on

**Pull request merged**